app/os/main.cpp

```diff
@@ -24,7 +25,9 @@ auto battery	  = leka::CoreBattery {PinName::BATTERY_VOLTAGE, charge_input};
 auto sd_blockdevice = SDBlockDevice {SD_SPI_MOSI, SD_SPI_MISO, SD_SPI_SCK};
 auto fatfs			= FATFileSystem {"fs"};
 
-auto rc = RobotController {sleep_timeout, battery};
+auto video_kit = VideoKit {};
```

Comment:
je préfère en un mot pour les kits, c'est plus joli à regarder

```suggestion
auto videokit = VideoKit {};
```